src/main/java/org/opensearch/knn/training/TrainingJobRunner.java

```diff
@@ -82,7 +83,9 @@ public void execute(TrainingJob trainingJob, ActionListener<IndexResponse> liste
         // the number of training jobs that enter this function. Although the training threadpool size will also prevent
         // this, we want to prevent this before we perform any serialization.
         if (!semaphore.tryAcquire()) {
-            throw new RejectedExecutionException("Unable to run training job: No training capacity on node.");
+            ValidationException exception = new ValidationException();
+            exception.addValidationError("Unable to run training job: No training capacity on node.");
```

Comment:
Is it possible to print node name?

Reply:
No, that information is not readily available.

Reply:
Should we log the error as ```log.info```?